addons/mail/models/update.py

```diff
@@ -116,10 +117,15 @@ def update_notification(self, cron_mode=True):
                 self.env['ir.config_parameter'].sudo().set_param('database.expiration_date', result['enterprise_info'].get('expiration_date'), ['base.group_user'])
                 self.env['ir.config_parameter'].sudo().set_param('database.expiration_reason', result['enterprise_info'].get('expiration_reason', 'trial'), ['base.group_system'])
                 self.env['ir.config_parameter'].sudo().set_param('database.enterprise_code', result['enterprise_info'].get('enterprise_code'), ['base.group_user'])
+                if result['enterprise_info'].get('database_already_linked_subscription_url'):
+                    enterprise_info = {
+                        'database_already_linked_subscription_url': result['enterprise_info'].get('database_already_linked_subscription_url'),
```

Comment:
Wouldn't it be better to store these values in `ir.config_parameter` just like we do above?